interface.html

```diff
@@ -58,7 +58,7 @@ <h4 class="panel-title trash-link">
             </ul>
           </div>
         </div>
-        <a href="https://help.fliplet.com/file-manager/" class="help-icon" target="_blank">
+        <a data-show-help-link href="https://help.fliplet.com/file-manager/" class="help-icon" target="_blank">
```

Comment:
Please remove the word `show` from the data attribute

Reply:
@squallstar fixed